scripts/docs_link_checker.py

```diff
@@ -0,0 +1,306 @@
+import glob
+import logging
+import os
+import re
+from typing import List
+
+import click
+import requests
+
+logger = logging.getLogger()
```

Comment:
Could we set the name using `logging.getLogger(__name__)`?

Reply:
Also kudos on all the logging 🙇🏽  I think we really need to improve here and this is a great example to use moving forward

Reply:
done!